test/server/lib/db/query.fixtures.js

```diff
@@ -53,32 +52,47 @@ FROM (
 ) AS counts
 GROUP BY name, dataset
 ORDER BY name~'^/Team' desc, name`
-  },
+	},
 
-  usageByUser: {
-    fromDateSQL: `SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER
+	usageByUser: {
+		fromDateSQL: `SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER
 FROM lev_audit
 WHERE date_time >= $(from)
 GROUP BY date, dataset, username
 ORDER BY date`,
-    fromToSQL: `SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER
+		fromToSQL: `SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER
 FROM lev_audit
 WHERE date_time >= $(from) AND date_time < $(to)
 GROUP BY date, dataset, username
 ORDER BY date`
-  },
+	},
 
-  searchTotals: {
-    totalCountSQL: `SELECT count(*)::INTEGER FROM lev_audit`,
-    todayCountSQL: `SELECT count(*)::INTEGER FROM lev_audit WHERE date_time >= $(from)`
-  },
+	searchTotals: {
+		totalCountSQL: `SELECT count(*)::INTEGER FROM lev_audit`,
+		todayCountSQL: `SELECT count(*)::INTEGER FROM lev_audit WHERE date_time >= $(from)`
+	},
 
-  searchTimePeriodByGroup: {
-    fromToGroupSQL: `SELECT count(*)::INTEGER
+	searchWithGroupFiltering: {
+		fromToGroupSQL: `SELECT count(*)::INTEGER
 FROM lev_audit
 WHERE date_time >= $(from) AND date_time < $(to) AND groups::TEXT ILIKE '%' || $(group) || '%'`,
-    gorupOnlySQL: `SELECT count(*)::INTEGER
+		multipleWithoutGroupsOnly: `SELECT count(*)::INTEGER
+FROM lev_audit
+WHERE date_time >= $(from) AND NOT (groups && $(withoutGroups))`,
+		singleWithoutGroupsOnly: `SELECT count(*)::INTEGER
```

Comment:
Why if there's a convention of naming things `blaBlaSQL`, do you have to go and be different @aarongill1 !?

Reply:
Because I'm a rebel...amended to conform.